msteams-platform/toolkit/add-capability.md

```diff
@@ -68,10 +68,11 @@ Apart from the capabilities your Teams app already have, you can choose to add d
 |Tabs with SPFx|None|
 |Tabs with Azure|Bots and messaging extensions|
 |Bots|Tabs|
-|Messaging extensions|Tabs|
-|Tabs and bots|None|
-|Tabs and messaging extensions|None|
-|Tabs, bots, and messaging extensions|None|
+|Messaging extensions|Tab and Bot|
+|Tabs and bots|Tab and Bot|
```

Comment:
one project can have 16 tabs, 1 bot and 1 message extension. So |Tabs and bots|Tab and Bot| is wrong, correct one should be |Tabs and bots|Tab and Message extension|

Reply:
Corrected